tx/textual/valuerenderer/valuerenderer.go

```diff
@@ -100,6 +101,18 @@ func (r Textual) GetValueRenderer(fd protoreflect.FieldDescriptor) (ValueRendere
 	}
 }
 
+// GetMessageValueRenderer is a specialization of GetValueRenderer for messages.
+// It is useful when the message type is discovered outside the context of a field,
+// e.g. when handling a google.protobuf.Any.
+func (r Textual) GetMessageValueRenderer(md protoreflect.MessageDescriptor) (ValueRenderer, error) {
```

Comment:
Should we standardize on either pointer or value receives for `Textual` in this file? I see 2 of both.

Reply:
Done.